libMBIN/Source/NMS/GameComponents/GcVehicleType.cs

```diff
@@ -6,7 +6,9 @@ namespace libMBIN.NMS.GameComponents
 	[NMS(Size = 0x4, GUID = 0xD3947F166802A50B, NameHash = 0x46812D05ED43A48D)]
     public class GcVehicleType : NMSTemplate
     {
-		public enum VehicleTypeEnum { Buggy, Bike, Truck, WheeledBike, Hovercraft, Submarine, Mech }
+        //Note: enum in exe shows Buggy, Bike, ... but GcVehicleGlobals.MBIN shows that the first entry is Bike and 2nd is Buggy
```

Comment:
I think we probably don't need this comment? It's good to have one in the vehicle globals but maybe not important here...

Reply:
You and I know it, but someone else may not and may think this is an error.  I would leave it there, it does explain the difference and points to GcVehicleGlobals in the event that NMS would correct that.